library/HelloSign/Client.php

```diff
@@ -1060,24 +1064,24 @@ public function updateApiApp($client_id, ApiApp $app)
     }
 
     /**
-     * Retrieves an API App with the given Client ID
+     * Retrieves a Bulk Send Job with the given Bulk Send Job ID
      *
-     * @param  String $id Client ID
-     * @return ApiApp
+     * @param  String $id Bulk Send Job ID
+     * @return BulkSendJob
      * @throws BaseException
      */
-    public function getApiApp($id)
+    public function getBulkSendJob($id)
```

Comment:
Did you intentionally replace the `getApiApp` function with `getBulkSendJob`? Why was `getApiApp` no longer needed?